src/data/queries/vetCenterLocationListing.ts

```diff
@@ -42,6 +53,67 @@ export const data: QueryData<
   return entity
 }
 
+// Single formatter function that handles all Vet Center variants
+const formatVetCenterVariant = (
```

Comment:
I expect to pare this down further as I figure out what information I actually need for the whole locations page.